drools-wb-webapp/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 JBoss Inc
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<jboss-deployment-structure xmlns="urn:jboss:deployment-structure:1.0">
+  <deployment>
+    <dependencies>
+
+      <!-- add JAXB processing -->
+      <module name="com.sun.xml.bind"/>
```

Comment:
`com.sun.xml.bind` is a EAP/WildFly private module and the application should not, in _ideal_ case, depend on those. It would be better to bundle this dependency directly into WEB-INF/lib (unless there is a other reason not to, which I am not aware of). In this case, it is not that important as we don't support `drools-wb-webapp` for user deployments.